app/models/member.rb

```diff
@@ -203,6 +204,12 @@ def groups
     return groups.values
   end
 
+  # We cannot enforce that a member is created with a calendar_id, but we can enforce that
```

Comment:
This sentence is a contradiction, might want to rewrite it because now I'm not sure if this sentence makes sense.